package.json

```diff
@@ -25,7 +26,15 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "build:lib": "json -I -f tsconfig.json -e 'this.compilerOptions.noEmit=false' && tsc --project tsconfig.json --declaration",
     "test": "react-scripts test",
+    "test:all": "react-scripts test --watchAll=false --passWithNoTests",
```

Comment:
please change to `test:ci`,
also please add "tsc &&" as prefix for this command,
there is an open issue for cra team, they basically ignore ts errors in .test files... so we need this prefix.

Reply:
But we are running `yarn typecheck` beforehand anyway.

Reply:
Understood, so nvm